server/server.js

```diff
@@ -34,11 +34,11 @@ app.post('/api/game', (req, res) => {
   const body = req.body;
```

Comment:
ticket requests that if "name" is not provided then the response should have a status of 400. can you test whether this works?